src/components/PlaceThumbnail.js

```diff
@@ -33,22 +36,30 @@ class PlaceThumbnail extends React.Component {
     const path = geoPath().projection(projection)
     const geoStates = feature(usa, usa.objects.units).features
     const meshed = mesh(usa, usa.objects.units, (a, b) => a !== b)
-    let active
-    if (usState !== 'washington-dc') {
-      active = geoStates.find(
-        s => s.properties.name === lookupUsa(usState).display,
-      )
+    const actives = []
+
+    if (placeType === 'region') {
+      const regionStates = lookupStatesByRegion(states.states, lookupRegionByName(region.regions, place).region_code)
+      Object.keys(regionStates).forEach(sr => {
+        actives.push(geoStates.find(
+          s => s.properties.name === regionStates[sr].state_name,
+        ))
+      });
+    } else if (place !== 'washington-dc') {
+      actives.push(geoStates.find(
+        s => lowerCase(s.properties.name) === lowerCase(place),
+      ))
     } else {
-      active = geoStates.find(s => s.id === 'US11')
+      actives.push(geoStates.find(s => s.id === 'US11'))
```

Comment:
what has an ID of US11?

Reply:
Washington DC. {"type":"Feature","id":"US11","properties":{"name":"District of Columbia"},"geometry":{...}}}